aiqleads/utils/logging.py

```
@@ -1,18 +1,64 @@
         )
-        handler.setFormatter(formatter)
-        logger.addHandler(handler)
+        console_handler.setFormatter(console_formatter)
+        logger.addHandler(console_handler)
+        
+        # File handler (log to data/logs directory)
+        try:
+            log_dir = os.path.join("aiqleads", "data", "logs")
+            os.makedirs(log_dir, exist_ok=True)
+            
+            today = datetime.now().strftime("%Y-%m-%d")
```

Ruff (0.8.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)